docs/templates/preprocessing/image.md

```diff
@@ -151,3 +153,38 @@ model.fit_generator(
         validation_data=validation_generator,
         nb_val_samples=800)
 ```
+
+Example of using it to transform images and masks together.
+
+```python
+
+# we create two instances with the same arguments
+data_gen_args = dict(
+    featurewise_center=True,
+    featurewise_std_normalization=True,
+    rotation_range=90.,
+    width_shift_range=0.1,
+    height_shift_range=0.1,
+    zoom_range=0.2
+)
+generator_i = ImageDataGenerator(**data_gen_args)
+generator_m = ImageDataGenerator(**data_gen_args)
+
+# We need to provide the same seed to the fit and flow methods
+seed = 1
+fake_y = np.zeroes(images.shape)
```

Comment:
If you don't need labels, you should configure the generator not to use labels (`class_mode=None`).